src/lib/integrations/SvelteFuse.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts" generics="T">
```

Comment:
we might wanna investigate to use server endpoints for searching - might scale better in the long run 👍🏻

Reply:
Was already thinking about this; will throw this in as a busy work task between Init stuff this week.